[2024-12-09T21:00:50.796+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-09T21:00:50.815+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.combine_sources manual__2024-12-09T21:00:41.632133+00:00 [queued]>
[2024-12-09T21:00:50.824+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.combine_sources manual__2024-12-09T21:00:41.632133+00:00 [queued]>
[2024-12-09T21:00:50.825+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-09T21:00:50.840+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): combine_sources> on 2024-12-09 21:00:41.632133+00:00
[2024-12-09T21:00:50.850+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=88) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-09T21:00:50.852+0000] {standard_task_runner.py:72} INFO - Started process 97 to run task
[2024-12-09T21:00:50.852+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb_complex', 'combine_sources', 'manual__2024-12-09T21:00:41.632133+00:00', '--job-id', '345', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb_complex.py', '--cfg-path', '/tmp/tmptfdh942_']
[2024-12-09T21:00:50.853+0000] {standard_task_runner.py:105} INFO - Job 345: Subtask combine_sources
[2024-12-09T21:00:51.170+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb_complex.combine_sources manual__2024-12-09T21:00:41.632133+00:00 [running]> on host 90de14a479be
[2024-12-09T21:00:51.241+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb_complex' AIRFLOW_CTX_TASK_ID='combine_sources' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T21:00:41.632133+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-09T21:00:41.632133+00:00'
[2024-12-09T21:00:51.242+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-09T21:00:51.243+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-09T21:00:51.243+0000] {logging_mixin.py:190} INFO - Current task name:combine_sources state:running start_date:2024-12-09 21:00:50.815964+00:00
[2024-12-09T21:00:51.244+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex and current dag run status:running
[2024-12-09T21:00:51.244+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-09T21:00:51.256+0000] {logging_mixin.py:190} INFO - Données combinées et sauvegardées sous /tmp/combined_data.json
[2024-12-09T21:00:51.257+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-09T21:00:51.266+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-09T21:00:51.267+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_mongodb_complex, task_id=combine_sources, run_id=manual__2024-12-09T21:00:41.632133+00:00, execution_date=20241209T210041, start_date=20241209T210050, end_date=20241209T210051
[2024-12-09T21:00:51.276+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-09T21:00:51.277+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-09T21:00:51.277+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex queued_at:2024-12-09 21:00:41.647542+00:00
[2024-12-09T21:00:51.278+0000] {logging_mixin.py:190} INFO - Task hostname:90de14a479be operator:PythonOperator
[2024-12-09T21:00:51.308+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-09T21:00:51.511+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-09T21:00:51.513+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
